src/ui/Facet/ValueElementRenderer.ts

```diff
@@ -62,8 +62,11 @@ export class ValueElementRenderer {
       title: l('ExcludeValueWithResultCount', this.caption, l('ResultCount', this.count, parseInt(this.count, 10))),
       className: 'coveo-facet-value-exclude',
       tabindex: 0,
-      role: 'button'
+      role: 'button',
+      'aria-hidden': 'true'
```

Comment:
Could this PR conflict with the work here? https://github.com/coveo/search-ui/pull/1585

Reply:
![Screen Shot 2020-09-01 at 9 20 15 AM](https://user-images.githubusercontent.com/4923043/91856297-62e74680-ec34-11ea-9865-53d0541b232e.png)
Maybe have aria-hidden on the svg itself?

Reply:
JF is right, I believe the `aria-hidden` should be on the SVG itself.